app/services/EventSinkService.scala

```diff
@@ -41,20 +29,18 @@ class EventSinkService {
   val queueName = Play.configuration.getString(EventSinkService.QUEUE_NAME_CONFIG_KEY)
     .getOrElse(EventSinkService.QUEUE_NAME_DEFAULT_VALUE)
 
-  def logEvent(category: String, metadata: JsValue) = {
-    Logger.info("eventsink.exchangename=" + exchangeName)
-    Logger.info("eventsink.queueName=" + queueName)
-
-    Logger.info("Submitting message to event sink exchange: " + Json.stringify(metadata))
-
-    //val message = EventSinkMessage(Instant.now().getEpochSecond, category, metadata)
-    messageService.submit(exchangeName, queueName, metadata, "fanout")
+  def logEvent(message: JsValue) = {
+    // Inject timestamp before logging the event
+    val event = message.as[JsObject] + ("created" -> Json.toJson(java.util.Date.from(Instant.now())))
+    Logger.info("Submitting message to event sink exchange: " + Json.stringify(event))
+    messageService.submit(exchangeName, queueName, event, "fanout")
```

Comment:
I think if we pass `""` or None here (not sure what the default is) it will not send it to a queue, but just to the exchange.

Reply:
The queueName is used as routing key further below 

```tempChannel.basicPublish(exchange, routing_key, null, message.toString.getBytes)```

It seemed fine when I ran the code with and without the queue.

Reply:
We will need to revisit this call. We are currently using the name of the queue as both the queue name and routing key. The meaning of the routing key for exchange is deepened on the the type of exchange. For example, in the current use of this function we are using fanout and that one doesn't use routing keys.